### DISCUSSION TITLE
How can i create msi file for python exe ?

### DISCUSSION THREAD

**Discussion by amarnathreddy0201** [wixtoolset/issues#7316] [Questions]
Hi ,
I try to use wix for  creating msi using python exe with visual studio 2022. i don't no how to use wix tool in vs 2022.

I try python version pip install WiX , in this case platform error.

1) If it possible with VS2022 to creating msi Suggest me.
2) With the python also ok for creating msi.

Thanks,
ANR .S

**Comment by robmen** [comment]
1. Yes. Install [HeatWave for VS2022](https://marketplace.visualstudio.com/items?itemName=FireGiant.FireGiantHeatWaveDev17).
2. I don't use Python so I don't know the answer to this.
